lib/jbuild

```diff
@@ -0,0 +1,8 @@
+(library
+ ((name        conduit)
+  (public_name conduit)
+  (wrapped false)
+  (preprocess (pps (ppx_sexp_conv)))
+  (modules (conduit conduit_trie resolver))
```

Comment:
Probably not the right time for this, but we should probably break compat at some point and set wrapped to true here. resolver is quit a common name to squat.